src/components/createAppointment/timePicker/timePicker.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react'
+import { PickerWrapper } from '../createAppointment.styles'
+import { Text } from '../createAppointment.styles'
+import { Listing, ListItem } from '../timePicker/timePicker.styles'
+
+const doctorsSchedule = [
+  '12:00 am',
+  '1:00 pm',
+  '2:00 pm',
+  '3:00 pm',
+  '4:00 pm',
+  '5:00 pm',
+  '6:00 pm',
+  '7:00 pm',
+  '8:00 pm',
+  '9:00 pm',
+]
+
+const TimePicker = ({ onChangeTimeData }) => {
+  const [time, setTime] = useState('')
+  const [active, setActive] = useState('')
```

Comment:
it is unused 'active' i think u can remove it

Reply:
removed